blocking-client/get-poetry.py

```diff
@@ -65,7 +65,8 @@ def get_poetry(address):
             sock.close()
             break
 
-        poem += data
+        # convert network stream to unicode for poem
```

Comment:
I think this comment is not needed, it's clear from the code what's happening.

Reply:
OK. pushed change.